### PR TITLE
Added search functionality for both medical and medicine

### PR DESCRIPTION
**Pull request by YadavGulshan**

### CODE REVIEW

File: backend/backend/urls.py [hunk 1]
```diff
@@ -1,18 +1,13 @@
-"""backend URL Configuration
+# pylint: disable=missing-module-docstring
+# 
+# Copyright (C) 2022 by YadavGulshan@Github, < https://github.com/YadavGulshan >.
```
Comment (rahul007-bit) [inline backend/backend/urls.py:3]: Kya baat kya baat

File: backend/pharmacy/api/serializers.py [hunk 2]
```diff
@@ -13,6 +23,12 @@ class MedicalSerializer(serializers.ModelSerializer):
     class Meta:
         model = Medical
         fields = '__all__'
+
+
+class MedicineSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Medicine
+        fields = '__all__'
```
Comment (rahul007-bit) [inline backend/pharmacy/api/serializers.py:31]: Yaha Kya ho raha hai

**Review by rahul007-bit — APPROVED** [review]

File: backend/pharmacy/api/serializers.py (hunk 2 above)
Comment (YadavGulshan) [reply backend/pharmacy/api/serializers.py:31]: Isme buss woh serializer ko bata raha h ki model Medicine ke saare fields ko serialize krna h api request me

Comment (YadavGulshan) [reply backend/pharmacy/api/serializers.py:31]: let's say Medicine model has `name`, `price` field... I'm just telling that utilize all the field, i.e both name and price and use them when giving a response or validate if every field is sent or not.

**Comment by YadavGulshan** [comment]
I think I should merge the req...

File: backend/pharmacy/api/serializers.py (hunk 2 above)
Comment (rahul007-bit) [reply backend/pharmacy/api/serializers.py:31]: ohhhhhh got it

**Review by rahul007-bit — COMMENTED** [review]
whats here?

**Comment by YadavGulshan** [comment]
> whats here?

I was trying something... But failed

**Pull request merged**